ethereumj-core/src/main/java/org/ethereum/manager/BlockLoader.java

```diff
@@ -125,7 +126,9 @@ private void blockWork(Block block) {
      * @return <code>true</code> if all blocks within all dumps have been successfully imported, <code>false</code> otherwise.
      */
     public boolean loadBlocks(Function<Path, DumpWalker> walkerFactory, Path... paths) {
-        Assert.notEmpty(paths, "There is nothing to import.");
+        if (ArrayUtils.isEmpty(paths)) {
+            throw new FatalEthereumException("There is nothing to import.");
```

Comment:
This doesn't look like fatal input to me.
It's just 0 input, 0 processed, everything fine.

Reply:
Yeah, you are right !
I changed this logic. Now the method returns `false` in such case.
According to this I fixed client's code in `Start` and `Harmony`.